src/runner/provider/qti.js

```diff
@@ -277,6 +277,14 @@ var qtiItemRuntimeProvider = {
                 done(renderingQueue);
             }, this.getLoadedClasses());
         });
+    },
+
+    /**
+     * Check if item is cleared
+     * @returns {boolean}
+     */
+    isCleared: function () {
+        return !this._item;
```

Comment:
**thought:** What is the purpose of the API, check the item is loaded? I wonder if the API is appropriate for the ItemRunner life-cycle.